basket/BABY_LOUIE/src/js/global.js

```diff
@@ -6,15 +6,16 @@
 import jQuery from 'jquery';
 
 (function ($, Drupal) {
-  "use strict";
+  // 'use strict';
+
   /**
    * Instantiate globalSite and run it through Drupal attachment.
    */
   Drupal.behaviors.globalSite = {
-    attach: function (context, settings) {
-      $(context).find('body').each(function() {
+    attach(context, settings) {
+      $(context).find('body').each(() => {
         // Do stuff
       });
-    }
+    },
```

Comment:
Does this comma need to be here?

Reply:
Standards say no "comma-dangle". Will be removed.